crowbar_framework/config/initializers/delayed_job_config.rb

```diff
@@ -14,12 +14,27 @@
 # limitations under the License.
 #
 
+module Delayed
+  class Worker
+    @@log_file = if Rails.env.production?
+      File.join(ENV["CROWBAR_LOG_DIR"], "background_jobs.log")
+    else
+      File.join(Rails.root, "log", "background_jobs.log")
+    end
+    @@logger = Logger.new(@@log_file)
```

Comment:
Basically because I need to access the log_file var which is set up as part of the environment, and I didnt want to check the env twice (one for the logger, one for the class log message)